.github/workflows/ci.yml

```diff
@@ -5,10 +5,10 @@ on:
   workflow_call:
   push:
     branches: ['main', 'develop']
-    paths-ignore: ['docs/**', 'docker/**', '*', '!pyproject.toml']
+    paths-ignore: ['docs/**', 'docker/**', '*', '!pyproject.toml', '!*.md']
```

Comment:
Don't we want this to be without the `!` here because the key is `paths-ignore`. So essentially right now we would explicitly not ignore any markdown files?

Reply:
Also i'm not sure about that but it might be `**.md` to catch them in all subdirectories?